src/lib/FreeC/Backend/Agda/Syntax.hs

```diff
@@ -153,10 +200,36 @@ ident = Ident . qname' . name
 
 -- | Hides the given expression.
 --
---   e ↦ {e}
+--   > e ↦ {e}
 hiddenArg_ :: Expr -> Expr
 hiddenArg_ = HiddenArg NoRange . unnamed
 
+-- | @if_then_else_@ from the base library.
+--
+--   > cond true false ↦ if cond then true else false
+ite :: Expr -> Expr -> Expr -> Expr
+ite cond true false =
+  RawApp NoRange [ident "if", cond, ident "then", true, ident "else", false]
+
+-- | @case_of_@ from the base library.
+--
+--   > disrc clauses ↦ case disrc of λ { clause₁ ; clause₂ ; … }
+caseOf :: Expr -> [LamClause] -> Expr
+caseOf discr alts =
+  RawApp NoRange [ident "case", discr, ident "of", ExtendedLam NoRange alts]
+
+-- | Smart constructor for @LamClause@s.
```

Comment:
Either link to `LamClause` using single quotes instead of at signs or describe what a `LamClause` is.

Reply:
I'd suggest to paraphrase `@LamClause@` in the subject line as follows.

```suggestion
-- | Smart constructor for a clause of a pattern matching lambda abstraction.
```

In the paragraph below `@LamClause@` could just be replaced by the word `clause`, for example.